.github/workflows/master.yml

```diff
@@ -24,3 +24,6 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose
+    - name: Run clippy
+      run: cargo clippy -- -D warnings
+      continue-on-error: true
```

Comment:
Can we remove this if #8 is merged?

Reply:
Yes, I overlooked it. It is removed now. 👍